packages/gatsby-plugin-utils/src/polyfill-remote-file/utils/url-generator.ts

```diff
@@ -29,7 +98,7 @@ export function generateFileUrl({
     })}/${filenameWithoutExt}${fileExt}`
   )
 
-  parsedURL.searchParams.append(ImageCDNUrlKeys.URL, url)
+  parsedURL.searchParams.append(getUrlParamName(), maybeEncryptUrl(url))
```

Comment:
Should we write something like this instead so it's encapsulated into one function instead of 2 functions that potentially have different if checks?

```suggestion
  appendUrlParam(parsedURL.searchParams, url);
```

Reply:
That makes sense for sure 👍